packages/terra-slide-group/tests/wdio/slide-group-spec.js

```diff
@@ -0,0 +1,62 @@
+/* global browser, Terra, before */
+describe('Slide Group', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-slide-group/slide-group/default-slide-group'));
+
+    Terra.should.matchScreenshot({ selector: '#SlideGroup' });
+    Terra.should.beAccessible();
+  });
+
+  describe('One Item', () => {
+    before(() => browser.url('/#/raw/tests/terra-slide-group/slide-group/one-item-slide-group'));
+
+    Terra.should.matchScreenshot({ selector: '#SlideGroup' });
+    Terra.should.beAccessible();
+  });
+
+  describe('Non Animated', () => {
+    before(() => browser.url('/#/raw/tests/terra-slide-group/slide-group/non-animated-slide-group'));
+
+    it('should increment', () => {
+      browser.click('#increment-1');
+
+      const screenshot = browser.checkElement('#SlideGroup');
```

Comment:
lets use `Terra.should.matchScreenshot({ selector: '#SlideGroup' });` outside of this `it` block